app/assets/scss/application.scss

```diff
@@ -1,3 +1,7 @@
+// Reset styles on core HTML elements that might be injected by eg a user's browser, so that we have a known clean base
```

Comment:
Sorry, should it be `that might be injected eg by a user's browser`?

Reply:
Either/or is fine.